css/src/components/form/radio.scss

```diff
@@ -0,0 +1,117 @@
+$radio-circle-size: 1.25em !default;
+$radio-dot-size: 0.625em !default;
+$radio-dot-checked-color: $primary !default;
+$radio-dot-color: $text-subtle !default;
+$radio-timing-function: $input-timing-function !default;
+$radio-duration: $input-transition-duration !default;
+$radio-animation: boop !default;
+$radios-spacing: 0.5em !default;
+
+@mixin radio-dot {
+	position: absolute;
+	inset: 0;
+	width: $radio-dot-size;
+	height: $radio-dot-size;
+	margin: auto;
+	border-radius: 50%;
+	background-color: $body-background;
+	content: '';
+	overflow: hidden;
+}
+
+@mixin radio-dot-checked {
+	background: $radio-dot-checked-color;
+}
+
+.radio {
+	display: inline-flex;
+	position: relative;
+	align-items: flex-start;
+	line-height: 1.25;
+	cursor: pointer;
+
+	.radio-dot {
+		display: inline-block;
+		position: relative;
+		inset-block-start: 0.0625em;
+		flex-shrink: 0;
+		width: $radio-circle-size;
+		height: $radio-circle-size;
+		border: 1px solid $text;
+		border-radius: $radius-rounded;
+		color: $radio-dot-color;
+		appearance: none;
+
+		&::before {
+			@include radio-dot;
+		}
+	}
+
+	.radio-label-text {
+		margin: 0 $radios-spacing;
+	}
+
+	legend + & {
+		margin-block-start: $radios-spacing;
+	}
+
+	.radios & {
```

Comment:
Should `radios` related code be placed in a separate file, similar to `.buttons`?

Reply:
Moved to a new file `radios.scss`